modules/miner/miner_test.go

```diff
@@ -272,17 +271,15 @@ func TestIntegrationStartupRescan(t *testing.T) {
 	oldHeight := mt.miner.persist.Height
 	oldTarget := mt.miner.persist.Target
 
-	// Corrupt the miner, the corruption should be fixed by the rescan.
+	// Corrupt the miner and verify that a rescan repairs the corruption.
 	mt.miner.persist.RecentChange[0]++
 	mt.miner.persist.Height += 500
 	mt.miner.persist.Target[0]++
-
-	// Call rescan and block until the scan is complete.
+	mt.cs.Unsubscribe(mt.miner)
```

Comment:
I forgot to update this test when I took unsubscribing out of the miner rescan function (not needed, as it only happens at startup before there is a successful subscription)